openapi.yaml

```
@@ -4084,6 +4317,220 @@ paths:
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: 'List of Hetzner server types.'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  properties: { id: { type: integer }, name: { type: string }, description: { type: string }, cores: { type: integer }, memory: { type: number }, disk: { type: integer }, prices: { type: array } }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)